docs/04-api-reference/front-commerce-core/defineExtension.mdx

```diff
@@ -117,3 +117,30 @@ Example:
   "translations": "./extensions/acme/translations"
 }
 ```
+
+### `unstable_lifecycleHooks`
+
+:::caution
+
+This feature is unstable and may change in the future.
+
+:::
+
+_Optional_
+
+Specifies the lifecycle hooks that should be executed by the application.
+
+Example:
+
+```js
+{
+  unstable_lifecycleHooks: {
+    onServerServicesInit: async (services, request, config) => {
+      // Do something with the services
+    },
+    onClientServicesInit: async (services, request, config) => {
+      // Do something with the services
+    }
```

Comment:
**issue:** [it doesn't exist](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/b8310856fc6a2daada9cfc83a421a7f670cdd3b4/packages/core/extensions/extensions.ts#L65)

**suggestion:** remove

Reply:
Removed.